input/vocabulary/valueset-ASAM Difference.json

```diff
@@ -0,0 +1,50 @@
+{
+    "resourceType" : "ValueSet",
+    "id" : "ASAMDifference",
+    "url" : "http://hl7.org/fhir/us/core/ValueSet/ASAMDifference",
+    "version" : "6.0.0",
+    "name" : "ASAMDifference",
+    "title" : "ASAM Difference",
+    "status" : "active",
+    "experimental" : false,
+    "date" : "2023-06-14",
+    "description" : "ASAM Difference details such as ASAM Level Change Reason Id and Reason Description.", 
+    "jurisdiction" : [{
+      "coding" : [{
+        "system" : "urn:iso:std:iso:3166",
+        "code" : "US"
+      }]
+    }],
+    "copyright" : "Used by permission of HL7 International, all rights reserved Creative Commons License",
+   "compose" : {
+      "include" : 
+      [
+             {
+          "system": "http://terminology.hl7.org/CodeSystem/",

```

Comment:
To use this, we will need to change the system, to create our own.
For now, use:  "http://signalbhn.org/fhir/CodeSystem/asam-difference"